### DISCUSSION TITLE
revert GE3 Fast texture back to old

### DISCUSSION THREAD

**Discussion by giraffemecrazy** [magknight/787aviatorsTracker#827] [Ideas]
I noticed that the 'old' GE3 fast spinning blades where a lot more immersive then the 2D ones we currently have, would it be possible to have either the new fan blades or atleast a 3d cone included in a future update? 

Thanks

**Comment by aeroniemi** [comment]
see https://github.com/magknight/787aviatorsTracker/issues/824 - we're not going to revert the changes.

**Reply by giraffemecrazy** [reply]
im aware, when i posted the suggestion it wasnt fixed yet.
